### PR TITLE
Completed the assessment! ON THE SAME DAY IT WAS GIVEN :O!

### PR DESCRIPTION
**Pull request by andeeisaacs**
Feeling pretty good about this week :) Thank you so much for taking the time to review my work.

### CODE REVIEW

File: code-challenges.js [hunk 1]
```diff
@@ -2,35 +2,56 @@
...
 // Expected output: true
 var hand2 = [5, 5, 3, 3, 4]
 // Expected output: false
+
+
+//Full house = two numbers have to match and then a different set of three numbers have to match. 
+
+//Creation of function that takes in an array of 5 numbers and determines whether or not the array is a “full house”. A full house is one pair and one three of a kind.
+
+
```
Comment (sjproctor) [inline code-challenges.js:48]: Love the approach, but it is a little fragile. It passes the given test cases, but fails with a test case of [2, 2, 2, 2, 4]

File: rails-questions.md [hunk 2]
```diff
@@ -1,27 +1,56 @@
...
-show -
+get - CRUD command to retrieve from the server
+library - The View page/file
+:id -The parameter identifier, which happens to be named id. 
+book - Our controller identifier
+show - the name of the method inside of our controller.
 
 3. Oops, I forgot to add a foreign key to my model. Describe the steps to remedy this mistake.
 
+Step 1, as described by Douglas Adams- DONT PANIC.
```
Comment (sjproctor) [inline rails-questions.md:19]: 😍

File: rails-questions.md [hunk 3]
```diff
@@ -1,27 +1,56 @@
...
 
+Step 7: Let rails work its magic. Once it has given you a success message, check your schema file to ensure your changes have taken effect. 
 
+Step 8: Give yourself a pat on the back, you just edited your schema without having to break the golden rule of never touching the file! WOW. You truly are a rails wizard. 
+
+
+
+4. What is the command to generate a Rails model for a person table with columns named shirt, pants, and shoes?
+ - rails generate model Person shirt:string pants:string shoes:string
+ - (hehehe, shoes:string... hehehehe. shoe string. heh.)
```
Comment (sjproctor) [inline rails-questions.md:48]: I have been laughing at this one all day!!!!

**Review by sjproctor — APPROVED** [review]
Dude, killing it. So happy for you.

**Comment by andeeisaacs** [comment]
Thank you again for taking the time to review my assessment and provide me some feedback! Means a lot! :D

**Pull request merged**